### PR TITLE
Reevaluating our color variables

### PR DESCRIPTION
**Pull request by paulkelley**
I took a look at our color variables which I thought were not ideal. I wanted to simplify what we had and focus on naming conventions that made more sense. I tried to limit variable usage as much as possible since it seemed I always had to weed out unneeded variables; $matte was always the biggest offender of this.

I also added variables for some social networks; I always seem to be looking these up.

### CODE REVIEW

**Comment by mbfitz** [comment]
I like this approach so much better. Also good call on adding the social variables! Looks good to me. ++1

File: css/styles.css [hunk 1]
```diff
@@ -476,27 +480,27 @@ input[type="submit"] {
   button:hover,
   input[type="reset"]:hover,
   input[type="submit"]:hover {
-    background-color: #5b646a;
-    border-color: #229AB7;
+    background-color: #666666;
+    border-color: #0c96dd;
     cursor: pointer; }
```
Comment (TommyChanthaboune) [inline css/styles.css]: Weird syntax here. Shouldn't the closing brace be on a new line ?

**Comment by TommyChanthaboune** [comment]
+1

**Comment by ghost** [comment]
Thanks for doing this - easier to read than the original. +1

On Wed, Jun 1, 2016 at 4:59 PM, Tommy Chanthaboune <notifications@github.com

> wrote:
> 
> +1
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/imarc/boilerplate/pull/47#issuecomment-223122873, or mute
> the thread
> https://github.com/notifications/unsubscribe/APDo5HWoTEGgPcQy0MV3YDitardeoMUTks5qHfKpgaJpZM4Ir9kQ
> .

## 

[image: iMarc Logo]

_Shawna O'Neal_ • Jr. Web Engineer • Imarc, A Digital Agency
shawna@imarc.com victoria@imarc.com • 978.462.8848

File: css/styles.css (hunk 1 above)
Comment (paulkelley) [inline css/styles.css]: That's happening a lot in the css file that got generated, I don't know why.

**Comment by khamer** [comment]
+1, I'm merging this in.

As a note, there's a lot of 'noise' in this pull request because
- We switched formatting styles for compiling styles.css, so braces aren't wrapped anymore (actually, this was first caused by a commit on April 21st, and this undoes it)
- We switched from uppercase hex colors to mostly lowercase hex colors. We should pick one format and stick with it.

As I said, I'm merging this in, we can clean up formatting/styles separately after, as these are both minor enough that it doesn't make sense to block merging.

**Comment by paulkelley** [comment]
Kevin the wrapped braces was a hot topic, I think everyone decided that no one cares.

As for hex colors, I vote lower case.

**Comment by ghost** [comment]
I also vote lowercase, it seems to be the most frequently used format
anyways.

On Fri, Jun 3, 2016 at 11:01 AM, Paul Kelley notifications@github.com
wrote:

> Kevin the wrapped braces was a hot topic, I think everyone decided that no
> one cares.
> 
> As for hex colors, I vote lower case.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> https://github.com/imarc/boilerplate/pull/47#issuecomment-223603504, or mute
> the thread
> https://github.com/notifications/unsubscribe/APDo5K91gg9jJsNFj0CTj8i5tjgFNfItks5qIEHmgaJpZM4Ir9kQ
> .

## 

[image: iMarc Logo]

_Shawna O'Neal_ • Jr. Web Engineer • Imarc, A Digital Agency
shawna@imarc.com victoria@imarc.com • 978.462.8848

**Comment by mbfitz** [comment]
I vote lowercase as well!

**Comment by khamer** [comment]
(Re: braces, it'll just be easier to see the differences in a commit and so on if we pick one way and stick with it. Personally, I think what you have is better than having wrapped braces, because if we ever did want to copy from the CSS file, it'd be more usable. The wrapped braces is the weird half-way between minified and readable that doesn't make a lot of sense to me.)

Lowercase hex is fine with me - if someone wants to make a PR for fixing formatting and updating the comb file, that'd be wonderful; if not, we'll get to it before the next 'release'.

**Comment by ghost** [comment]
Dibs on hex case swapping and comb file.

On Fri, Jun 3, 2016 at 11:30 AM, Kevin Hamer notifications@github.com
wrote:

> (Re: braces, it'll just be easier to see the differences in a commit and
> so on if we pick one way and stick with it. Personally, I think what you
> have is better than having wrapped braces, because if we ever did want to
> copy from the CSS file, it'd be more usable. The wrapped braces is the
> weird half-way between minified and readable that doesn't make a lot of
> sense to me.)
> 
> Lowercase hex is fine with me - if someone wants to make a PR for fixing
> formatting and updating the comb file, that'd be wonderful; if not, we'll
> get to it before the next 'release'.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> https://github.com/imarc/boilerplate/pull/47#issuecomment-223611430, or mute
> the thread
> https://github.com/notifications/unsubscribe/APDo5EUG1lfh2jjN5onpxZdL3Hjfbtx8ks5qIEiegaJpZM4Ir9kQ
> .

## 

[image: iMarc Logo]

_Shawna O'Neal_ • Jr. Web Engineer • Imarc, A Digital Agency
shawna@imarc.com victoria@imarc.com • 978.462.8848

**Pull request merged**